docs/balena-fleet-deployment.md

```diff
@@ -0,0 +1,25 @@
+# Deploying on a Balena fleet
```

Comment:
@vpetersson, here's a sneak peak of the new docs that I'm currently working on. Most of the steps will rely on balenaCloud CLI. I used this [old article](https://blog.balena.io/deploy-free-digital-signage-software-screenly-ose/) as one of my references.